server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -93,14 +98,17 @@ public class FeatureFlags {
 
     public static final Setting<Boolean> PLUGGABLE_CACHE_SETTING = Setting.boolSetting(PLUGGABLE_CACHE, false, Property.NodeScope);
 
+    public static final Setting<Boolean> PROTOBUF_SETTING = Setting.boolSetting(PROTOBUF, false, Property.NodeScope, Property.Dynamic);
```

Comment:
PROTOBUF_TRANSPORT_SETTING

Reply:
I think this variable and the name of the setting should match our intent. Won't we be rolling out the search API with protobuf support, then another API, etc.? What's the setting that will live the longest?

Either way this variable and the setting string should match I think.

The most generic one:

```
PROTOBUF = opensearch.experimental.feature.protobuf.enabled
PROTOBUF_SETTING
```

Search specific.

```
PROTOBUF_SEARCH = opensearch.experimental.feature.protobuf.search.enabled
PROTOBUF_SEEARCH_SETTING
```

I'm good with either, but I prefer (1) because I don't think we'll ever want to have 2 APIs with protobuf support enabled/disabled separately. I could totally be wrong, too.